requirements.txt

```diff
@@ -7,3 +7,5 @@ numpy>=1.17.*
 matplotlib>=3.1.*
 torchvision==0.8.2
 mlflow==1.13.1
+azure-storage-blob==2.1.0
+locust==1.4.1
```

Comment:
which image uses it? :/

Reply:
No image does, I added this to build my local virtualenv for testing, but forgot to remove.